static/admin/config.yml

```
@@ -40,8 +40,13 @@ collections:
               - { name: "locale", widget: "select", <<: *locales }
               - { name: "headline", widget: "string" }
               - { name: "subHeadline", widget: "string" }
-              - { name: "metaTitle", widget: "string" }
               - { name: "content", widget: "markdown" }
+              - name: "seo"
```

Would be a better I guess.
```suggestion
              - name: "meta"
```